src/main/java/edu/greenblitz/tobyDetermined/subsystems/Dashboard.java

```diff
@@ -19,23 +19,7 @@ public static Dashboard init() {
 	
 	@Override
 	public void periodic() {
-		SmartDashboard.putNumber("pigeon angle", SwerveChassis.getInstance().getPigeonGyro().getYaw());
```

Comment:
are you sure?

Reply:
why resolve? lets not delete the dashboard values if they are unrelated to the code change, unless you absolutely know for a fact we don't and won't need them